ecr-credential-provider.spec

```diff
@@ -0,0 +1,141 @@
+# Built from this template:
+# https://github.com/openshift/release/blob/master/tools/hack/golang/package.spec
+#
+# See also: https://github.com/openshift/kubernetes/blob/master/openshift.spec
+
+
+# This is a template package spec that will support Go builds following the
+# OpenShift conventions. It expects a set of standard env vars that define the
+# Git version being built and can also handle multi-architecture Linux builds.
+# It has stubs for cross building.
+
+
+#debuginfo not supported with Go
+%global debug_package %{nil}
+
+# modifying the Go binaries breaks the DWARF debugging
+%global __os_install_post %{_rpmconfigdir}/brp-compress
+
+# %commit and %os_git_vars are intended to be set by tito custom builders
+# provided in the .tito/lib directory. The values in this spec file will not be
+# kept up to date.
+%{!?commit: %global commit HEAD }
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+# os_git_vars needed to run hack scripts during rpm builds
+%{!?os_git_vars: %global os_git_vars OS_GIT_VERSION='' OS_GIT_COMMIT='' OS_GIT_MAJOR='' OS_GIT_MINOR='' OS_GIT_TREE_STATE='' }
+
+%if 0%{?skip_build}
+%global do_build 0
+%else
+%global do_build 1
+%endif
+%if 0%{?skip_prep}
+%global do_prep 0
+%else
+%global do_prep 1
+%endif
+
+%if 0%{?fedora} || 0%{?epel}
+%global need_redistributable_set 0
+%else
+# Due to library availability, redistributable builds only work on x86_64
+%ifarch x86_64
+%global need_redistributable_set 1
+%else
+%global need_redistributable_set 0
+%endif
+%endif
+%{!?make_redistributable: %global make_redistributable %{need_redistributable_set}}
+
+#
+# Customize from here.
+#
+
+%global golang_version 1.21.0
+%{!?version: %global version 0.0.1}
+%{!?release: %global release 1}
+%global package_name ecr-credential-provider 
+%global product_name ecr-credential-provider
```

Comment:
Is this what I would `dnf install`?

Reply:
I think this gets rewritten by brew

The actual spec file we're building from is: https://pkgs.devel.redhat.com/git/rpms/ose-aws-ecr-image-credential-provider/tree/ecr-credential-provider.spec?h=rhaos-4.16-rhel-8#n57 

and seems to ignore these two .... and rewrites to  ose-aws-ecr-image-credential-provider that matches the distgit repo